packages/element-app/src/components/SidetreeTransaction/SidetreeTransaction.js

```diff
@@ -19,15 +16,16 @@ import IconButton from '@material-ui/core/IconButton';
 import DoneAll from '@material-ui/icons/DoneAll';
 import VerifiedUser from '@material-ui/icons/VerifiedUser';
 import Receipt from '@material-ui/icons/Receipt';
+import ThumbsUpDownIcon from '@material-ui/icons/ThumbsUpDown';
 import ExpandMore from '@material-ui/icons/ExpandMore';
 import Link from '@material-ui/icons/Link';
 import LocalActivity from '@material-ui/icons/LocalActivity';
 import Forward from '@material-ui/icons/Forward';
 
-const getBlockExplorerUrl = (blockHash, blockchain, network) => {
+const getBlockExplorerUrl = (suffix, blockchain, network) => {
```

Comment:
I'm not a huge fan of encoding this data in a path, i'd rather build this suffix in the function, from the arguments... this is a tight coupling to etherscan.